src/m365/planner/commands/plan/Plan.ts

```diff
@@ -1,19 +1,17 @@
 export interface Plan {
```

Comment:
Rather than introducing our own type, can we reuse here the type exposed by the MS Graph types package?

Reply:
I'll check it out.

Reply:
@waldekmastykarz I changed my custom type with the MS Graph PlannerPlan type, but  `@odata.etag` property is missing here. How can I fix this issue?

![image](https://user-images.githubusercontent.com/36161889/153957037-96faaef5-f97a-4f15-b1f8-57c100eaa129.png)

Reply:
Let's define a custom type named `ExtendedPlannerPlan` and add `@odata.etag` property to it.

Reply:
Done